### PR TITLE
Add samples folder

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

**Review by hhariri — APPROVED** [review]
There are some grammar mistakes, but since it's a verbatim copy, I'm going to ignore those as this is about the new folder. We'll revisit those later. Thanks.

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Samples
+
+This sections contains samples for different cases of using Exposed.
```
Comment (joc-a) [inline samples/README.md:3]:
```suggestion
This section contains samples for different cases of using Exposed.
```

**Review by joc-a — APPROVED** [review]
Left a few nit-picky comments

**Comment by e5l** [comment]
@joc-a thanks!

**Pull request merged**